.github/ci-bionic/after_make.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh -l
+
+set -x
+
+# Install (needed for some tests)
+make install
+
+Xvfb :1 -screen 0 1280x1024x24 &
```

Comment:
are we sure that :1 is alway the display in use?

Reply:
I just copied what worked here: https://github.com/ignitionrobotics/ign-rendering/pull/115

And it looks like it also worked here, now there's only one test failure left.

Reply:
trying to relax the check in e0be114 . Let's see how it goes